config/crd/bases/discovery.open-cluster-management.io_discoveredclusters.yaml

```diff
@@ -160,6 +162,8 @@ spec:
                 type: string
               cloudProvider:
                 type: string
+              rhocmClusterId:
+                type: string
```

Comment:
You are providing `rhocmClusterId` twice. I believe this one should be `ocpClusterId`.

Reply:
oh nope, I just forgot to add it entirely :sweat_smile: